test/unit/basic.js

```diff
@@ -0,0 +1,268 @@
+module( "basic", {
+	teardown: moduleTeardown
+} );
+
+if ( jQuery.ajax ) {
+test( "ajax", function( assert ) {
+	expect( 4 );
+
+	var done = assert.async(),
+		requestsLeft = 3;
+
+	jQuery.ajax({
+		type: "GET",
+		url: url( "data/name.php?name=foo" ),
+		success: function( msg ) {
+			assert.strictEqual( msg, "bar", "Check for GET" );
+			requestsLeft--;
+			if ( !requestsLeft ) {
+				done();
```

Comment:
You don't need to do the tracking yourself; just use `assert.async` multiple times (cf. [test/unit/deferred.js](https://github.com/jquery/jquery/blob/1c59b308d201d6dd0f0aed2ad0256d01b9f68047/test/unit/deferred.js#L411-L421)).